### PR TITLE
meta: redirect to active repo

### PR DESCRIPTION
**Pull request by refack**
Refs: https://github.com/nodejs/node/issues/14662

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
May be we could also mention `nodejs/help` with distinction described? It would be a bit harsh if somebody is redirected twice with an issue like https://github.com/nodejs/node-v0.x-archive/issues/26596

**Review by vsemozhetbyt — APPROVED** [review]

**Review by gibfahn — APPROVED** [review]

**Comment by refack** [comment]
Anyone know what's the policy for landing in this repo?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Thank you for your pull request. This repository has been archived.
+Please open the PR in https://github.com/nodejs/node/compare
```
Comment (bnoordhuis) [inline .github/PULL_REQUEST_TEMPLATE.md]: Just `https://github.com/nodejs/node`?  The compare view won't work if the changes are in a fork of this repo.

**Review by bnoordhuis — APPROVED** [review]
LGTM with a suggestion.

@refack You can just land it, nothing out of the ordinary.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (refack) [reply .github/PULL_REQUEST_TEMPLATE.md]: Went with https://github.com/nodejs/node/pulls for as much focusing as possible 🤷‍♂️

**Review by Fishrock123 — APPROVED** [review]
Some bolding wouldn't hurt, doesn't matter too much though.

**Comment by Fishrock123** [comment]
Also yes, there is no rules for landing stuff here other than "usually don't except for redirection purposes" :P

**Comment by refack** [comment]
> Some bolding wouldn't hurt, doesn't matter too much though.

since it shows up as monotype, best we can do is some ASCII art

<sub>screenshot of node's new issue form</sub>
image

**Pull request merged**